services/QuillLMS/engines/evidence/app/views/layouts/evidence/research/gen_ai/application.html.erb

```diff
@@ -1,6 +1,6 @@
 <html lang='en'>
   <title>Evidence GenAI Experiments</title>
-  <%= stylesheet_link_tag 'evidence/research/gen_ai/application', media: 'all', 'data-turbolinks-track': 'reload' %>
+  <%= stylesheet_link_tag 'evidence/research/gen_ai/application', media: 'all' %>
```

Comment:
@brendanshean  `stylesheet_link_tag` is deprecated since the Vite migration. Let's discuss how we can port this to Vite. Should be easy

Reply:
I'll move the application.css to the main app.  It feels weird to have it in the main app, but after discussing with Peter, I would agree that having a vite pipeline just for the file seems silly.

Eventually that UI will be refactored into React and then we can get rid of that weird wrinkle by removing the file altogether.